.github/workflows/release-please.yml

```diff
@@ -0,0 +1,34 @@
+name: Release-Please
+
+# triggered when tests complete on main branch
+on:
+  workflow_run:
+    workflows:
+      - "Test"
+    branches:
+      - main
+    types:
+      - completed
+
+jobs:
+  refused:
+    # Refuse if tests failed
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'failure'}}
+    steps:
+      - id: refused_step
+        run: exit 1
+
+  release-please:
+    # Automate releases with Conventional Commit Messages as Pull Requests are merged into "main" branch
+    name: Prepare next release
+    runs-on: ubuntu-latest
+    # release only if tests pass
+    if: ${{ github.event.workflow_run.conclusion == 'success'}}
+    steps:
+      - uses: GoogleCloudPlatform/release-please-action@v2
+        with:
+          token: ${{ secrets.GH_TOKEN }}
+          release-type: python
+          package-name: gh_action_test
```

Comment:
Package name is maybe wrong here. This name will be used as prefix in the release